development/docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
     healthcheck:
       interval: "30s"
       timeout: "10s"
-      start_period: "2m"
+      start_period: "5m"
```

Comment:
FYI Nautobot core uses `10m` currently just to be overly safe.

Reply:
5m seems to be fine, locally and for the CI as well. I'll keep an eye on it, in case of issues I will increase it.